src/stores/security-browser-capabilities-detection.md

```diff
@@ -7,7 +7,7 @@ Like most websites and applications on the Internet, Adobe Commerce and Magento
 - If the browser’s privacy settings disallow cookies, you can configure the system to automatically redirect them to the [Enable Cookies]({% link cms/pages-core.md %}) page, which explains how to make the recommended settings with most browsers.
 - If the browser’s privacy settings disallow JavaScript, you can configure the system to display the following message above the header of every page.
 
-For technical information, see [Supported browsers][1]{:target="_blank"} in the developer documentation.
+For technical information, see [Supported browsers]({{ site.devdocs_url }}/install-gde/system-requirements_browsers.html#supported-browsers) in the developer documentation.
```

Comment:
The install guide is versioned on devdocs, to it also needs that part of the URL

```suggestion
For technical information, see [Supported browsers]({{ site.devdocs_url }}/guides/v{{ site.version }}/install-gde/system-requirements.html#supported-browsers) in the developer documentation.
```

Reply:
Hi @leslietilling , thank you for reviewing. I have implemented your suggestion, could you please approve this PR?